src/index.html

```diff
@@ -8,6 +8,49 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="container">
+      <div class="stars stars--0">
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+      </div>
+      <div class="stars stars--1">
+        <div class="stars__star yellow"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+      </div>
+      <div class="stars stars--2">
+        <div class="stars__star yellow"></div>
+        <div class="stars__star yellow"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+      </div>
+      <div class="stars stars--3">
+        <div class="stars__star yellow"></div>
+        <div class="stars__star yellow"></div>
+        <div class="stars__star yellow"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+      </div>
+      <div class="stars stars--4">
+        <div class="stars__star yellow"></div>
+        <div class="stars__star yellow"></div>
+        <div class="stars__star yellow"></div>
+        <div class="stars__star yellow"></div>
+        <div class="stars__star"></div>
+      </div>
+      <div class="stars stars--5">
```

Comment:
what if i change modifier on `stars--2` here
lefts 5 stars, but should change to 2